### PR TITLE
cc33xx-fw: add debian/ and support files

### PR DESCRIPTION
**Pull request by jsuhaas22**
cc33xx-fw package installs firmware binaries required to get WIFI to work on TI devices.

The copyright is [TI-IMG](https://git.ti.com/cgit/arago-project/meta-ti/tree/meta-ti-bsp/licenses/TI-IMG?h=scarthgap). Have manually edited some fields since debmake's output was wrong (haven't touched license text).

This is the last WIFI component, for now. Based on recipe:
https://git.ti.com/cgit/arago-project/meta-ti/commit/?h=scarthgap&id=6f750ff4a9a0dcde63dd1e44beb25b6fb7c0fb59

### CODE REVIEW

File: cc33xx-fw/suite/trixie/debian/cc33xx-fw.install [hunk 1]
```diff
@@ -0,0 +1 @@
+*.bin /lib/
```
Comment (glneo) [inline cc33xx-fw/suite/trixie/debian/cc33xx-fw.install]: Should this be `/lib/firmware`?

File: cc33xx-fw/suite/trixie/debian/control [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+Maintainer: Suhaas Joshi <s-joshi@ti.com>
+Build-Depends: debhelper-compat (= 13)
+Standards-Version: 4.6.1
+Homepage: https://git.ti.com/git/cc33xx-wlan/cc33xx-fw
+Rules-Requires-Root: no
+#Vcs-Git: https://salsa.debian.org/debian/cc33xx-fw.git
+#Vcs-Browser: https://salsa.debian.org/debian/cc33xx-fw
+
+Package: cc33xx-fw
+Architecture: arm64
```
Comment (glneo) [inline cc33xx-fw/suite/trixie/debian/control]: If this is firmware it should not be Architecture dependent, `All`.

File: cc33xx-fw/suite/trixie/debian/rules [hunk 3]
```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/make -f
+
+%:
+	dh $@
+
+#override_dh_auto_install:
```
Comment (glneo) [inline cc33xx-fw/suite/trixie/debian/rules]: Why all the commented out lines left in?

File: cc33xx-fw/suite/trixie/debian/source/patch-header [hunk 4]
```diff
@@ -0,0 +1,22 @@
+The automatically generated patch puts this free form text on top of it.
```
Comment (glneo) [inline cc33xx-fw/suite/trixie/debian/source/patch-header]: This file looks like some leftover auto-generated thing, this needed?

File: cc33xx-fw/suite/trixie/debian/cc33xx-fw.install (hunk 1 above)
Comment (jsuhaas22) [reply cc33xx-fw/suite/trixie/debian/cc33xx-fw.install]: It should be `/lib/firmware/ti-connectivity/`. Fixed.

**Comment by jsuhaas22** [comment]
Addressed your comments @glneo 
Not sure how I let these auto-generated files and comments in. Have removed them now.

**Comment by glneo** [comment]
Looks much better, thanks.

**Review by glneo — APPROVED** [review]

**Review by cshilwant — APPROVED** [review]

**Pull request merged**